nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -285,7 +273,13 @@ class RelPositionalEncoding(PositionalEncoding):
     """
 
     def __init__(self, d_model, dropout_rate, max_len=5000, xscale=None, dropout_rate_emb=0.0):
-        super().__init__(d_model, dropout_rate, max_len, xscale=xscale)
+        super().__init__(
+            d_model,
+            dropout_rate,
+            max_len,
+            xscale=xscale,
+            positions=torch.arange(max_len - 1, -max_len, -1, dtype=torch.float32),
```

Comment:
positions does not exist in the class this one inherited from.

Reply:
Why, it is there in PositionalEncoding

Reply:
Oh I see you added that to PositionalEncoding too.